services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from db.models import MovieSession, Movie, CinemaHall
+from django.db.models.query import QuerySet
+from typing import Optional
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet[MovieSession]:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime.date = None
```

Comment:
If the default value is None - please, use Optional[]

Reply:
done